pkg/assets/apps.go

```diff
@@ -55,6 +57,10 @@ func (d *dpApplier) Reader(objBytes []byte, render RenderFunc, params RenderPara
 		panic(err)
 	}
 	d.dp = obj.(*appsv1.Deployment)
+	if d.dp.Spec.Template.ObjectMeta.Annotations == nil {
+		d.dp.Spec.Template.ObjectMeta.Annotations = make(map[string]string)
+	}
+	d.dp.Spec.Template.ObjectMeta.Annotations["operator.openshift.io/force"] = uuid.NewString()
```

Comment:
Do we want this approach for all Deployments/DaemonSets we have or only the selected ones? ovn-k for example

Reply:
Maybe we want a flag to this function? Which is the more common case?

Reply:
Turns out this also flakes. I have seen an execution where the old pod had a deletion timestamp (but was not deleted yet and was signaling readiness) and the new one was ready as well. This failed greenboot because it does not expect 2 pods from different deployment/daemonset versions.
Going to try the old way again about restarting the pod from the test until we figure out how to do this gracefully from within MicroShift (or its components).